website/data/docs-nav-data.json

```diff
@@ -1437,17 +1437,17 @@
         "path": "upgrading/plugins"
       },
       {
-        "title": "Upgrade to 1.9.0",
-        "path": "upgrading/upgrade-to-1.9.0",
+        "title": "Upgrade to 1.9.x",
+        "path": "upgrading/upgrade-to-1.9.x",
         "hidden": true
```

Comment:
@mladlow I wonder if we should remove this attribute. Otherwise, the docs might not show when 1.9.x is versioned.

Reply:
I might leave it out of this PR since I want to keep the result from the diff the same from the resulting 3 cherry-picks that I did.